frontend/src/components/crud/Update.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { JSONSchema7 } from "json-schema";
+import { Form } from "@rjsf/bootstrap-4";
+import JSONPretty from "react-json-pretty";
+import EntityAPIClient from "../../APIClients/EntityAPIClient";
+
+enum EnumField {
+  "A",
+  "B",
+  "C",
+  "D",
+}
+
+type EntityResponse = {
+  id: string | number;
+  stringField: string;
+  intField: number;
+  stringArrayField: string[];
+  enumField: EnumField;
+  boolField: boolean;
+};
+
+const schema: JSONSchema7 = {
+  title: "Create Entity",
+  description: "A simple form to test creating an entity",
+  type: "object",
+  required: [
+    "id",
+    "stringField",
+    "intField",
+    "stringArrayField",
+    "enumField",
+    "boolField",
+  ],
+  properties: {
+    id: {
+      type: "string",
+      title: "entity id",
+      default: "2017",
+    },
+    stringField: {
+      type: "string",
+      title: "String Field",
+      default: "UW Blueprint",
+    },
+    intField: {
+      type: "integer",
+      title: "Integer Field",
+      default: 2017,
+    },
+    stringArrayField: {
+      type: "array",
+      items: {
+        type: "string",
+      },
+      title: "String Array Field",
+      default: [],
+    },
+    enumField: {
+      type: "string",
+      enum: ["A", "B", "C", "D"],
+      title: "Enum Field",
+      default: "A",
+    },
+    boolField: {
+      type: "boolean",
+      title: "Boolean Field",
+      default: true,
+    },
+  },
+};
+
+const uiSchema = {
+  boolField: {
+    "ui:widget": "select",
+  },
+};
+
+const UpdateForm = () => {
+  const [data, setData] = useState<EntityResponse | undefined>(undefined);
+
+  if (data) {
+    return (
+      <div>
+        <JSONPretty id="json-pretty" data={data} />
+      </div>
+    );
+  }
+
+  const onSubmit = async ({ formData }: { formData: any }) => {
+    const entityData: any = JSON.parse(JSON.stringify(formData));
```

Comment:
Why is the JSON parse needed here but not in the create component? Not super important but I think a cleaner way is to convert like this instead of stringifying and then parsing

```ts
let object = {};
formData.forEach((value, key) => object[key] = value);
```

Also would prefer not to type `entityData` as `any` unless absolutely necessary, I think it should be
```ts
EntityRequest & { id: string }
```